.github/workflows/buid_and_release.yml

```diff
@@ -13,15 +13,18 @@ on:
 jobs:
   build_and_release:
     runs-on: ubuntu-latest
+#    container: obolibrary/odkfull:v1.4.1
+#    container: obolibrary/odkfull:dev
+    container: obolibrary/odkfull:latest
```

Comment:
Always use a version to minimise reasons of failure.

Reply:
@matentzn Yeah I agree with that. I was thinking I should probably change that too. Glad you agree!